src/Components/RecsListTable/RecsListTable.spec.ct.js

```diff
@@ -209,18 +291,98 @@ describe('successful non-empty recommendations list table', () => {
   });
 
   describe('defaults', () => {
+    // TODO enhance tests See ClustersListTable
+
+    it(`shows maximum ${DEFAULT_ROW_COUNT} recommendations`, () => {
+      // TODO get a function like checkRowCounts with expandable rows
+      cy.get('table')
+        .find('[data-ouia-component-type="PF4/TableRow"]') // TODO use ROW from components module
+        .find(`td[data-label="Name"]`)
+        .should('have.length', DEFAULT_DISPLAYED_SIZE);
+      expect(window.location.search).to.contain('limit=20'); // TODO do not hardcode value
+    });
+
     it('default sort by total risk', () => {
+      const column = 'Total risk';
       // TODO do not use ROW but Table and th. See AffectedClustersTable
       cy.get(ROW)
         .children()
         .eq(0)
-        .find('td[data-label="Total risk"]')
+        .find(`td[data-label="${column}"]`)
         .contains('Critical'); // TODO do not use value hardcoded. Use class as in AffectedClustersTable
+      // TODO use columnName2UrlParam once !238 is merged
+      // expect(window.location.search).to.contain(`sort=-${columnName2UrlParam(column)}`);
       expect(window.location.search).to.contain('sort=-total_risk');
     });
+
+    it(`pagination is set to ${DEFAULT_ROW_COUNT}`, () => {
+      cy.get('.pf-c-options-menu__toggle-text')
+        .find('b')
+        .eq(0)
+        .should('have.text', `1 - ${DEFAULT_DISPLAYED_SIZE}`); // TODO do not hardcode value
+    });
+
+    it('reset filters button is displayed', () => {
+      cy.get('button').contains('Reset filters').should('exist');
+    });
   });
 
-  describe('pagination', () => {});
+  describe('pagination', () => {
+    it('shows correct total number of recommendations', () => {
+      checkPaginationTotal(filterData(DEFAULT_FILTERS).length);
+    });
+
+    it('values are expected ones', () => {
+      checkPaginationValues(PAGINATION_VALUES);
+    });
+
+    it('can change page limit', () => {
+      // FIXME: best way to make the loop
+      cy.wrap(PAGINATION_VALUES).each((el) => {
+        changePagination(el).then(() =>
+          // TODO should this be nested. Also the other check below?
```

Comment:
I believe so, if you don't nest it, then there can be an error/interface not updated as expected after the click. Found this article: https://www.webiny.com/blog/things-to-avoid-when-writing-cypress-tests "If you need to do something after a command has been completed, you can do so with the help of the cy.then command. It will guarantee that only after the previous command finishes, the next will run."

Reply:
Thanks @gkarat